tests/network/sriov.go

```diff
@@ -793,12 +793,15 @@ func createSRIOVVmiOnNode(nodeName, networkName, cidr string) (*v1.VirtualMachin
 
 	// manually configure IP/link on sriov interfaces because there is
 	// no DHCP server to serve the address to the guest
-	vmi := newSRIOVVmi([]string{networkName}, cloudInitNetworkDataWithStaticIPsByMac(networkName, mac.String(), cidr))
+	vmi := newSRIOVVmi(
+		[]string{networkName},
+		cloudInitNetworkDataWithStaticIPsByMac(networkName, mac.String(), cidr),
+		libvmi.WithNodeAffinityFor(nodeName),
+	)
 	const secondaryInterfaceIndex = 1
 	vmi.Spec.Domain.Devices.Interfaces[secondaryInterfaceIndex].MacAddress = mac.String()
 
-	vmi = tests.CreateVmiOnNode(vmi, nodeName)
-
+	vmi = tests.RunVMI(vmi, 90)
```

Comment:
Same, I do not think we need a retry for the creation. This should be now a simple client "create" call.